migration/migrator/migrations/master/20240805165505_user_verification_code.py

```diff
@@ -0,0 +1,35 @@
+"""Migration for the Submitty master database."""
+
+
+def up(config, database):
+    """
+    Run up migration.
+
+    :param config: Object holding configuration details about Submitty
+    :type config: migrator.config.Config
+    :param database: Object for interacting with given database for environment
+    :type database: migrator.db.Database
+    """
+
+    database.execute('''
+        CREATE TABLE if not exists public.unverified_users (
+            user_id character varying NOT NULL,
```

Comment:
Should we specify a maximum user ID length?

Reply:
Currently it doesn't look like the database enforces a maximum user ID length in the users or courses_users table, but in the SAML mapped users, and the sessions database, it has a max of 255 characters. Should I add one ?

Reply:
I think a max of 255 is more than reasonable for a user ID.  In fact, something much shorter like 50 characters would probably be sufficient too.